app/styles/components/events/view/export.scss

```diff
@@ -32,3 +32,11 @@
 .url {
   padding: 0 1em !important;
 }
+
+.copy-url {
```

Comment:
just remove this class @harsh-jindal

Reply:
@CosmicCoder96 Text will still overflow If i remove that class. Instead I have added media breakpoints to handle overflow.

Reply:
@harsh-jindal is the text-overflow being caused by the changes you made or is it a pre-existing issue?

Reply:
@CosmicCoder96 This issue was not there was before as class="ui segment secondary" took care of it. But #copy-button causes the overflow issue. That is why I have to explicitly define a class.